com.unity.cinemachine/Samples~/3D Samples/FreeLook Deoccluder.unity

```diff
@@ -212,7 +212,8 @@ Transform:
   m_LocalPosition: {x: 0, y: 1.5035291, z: -4.7685847}
```

Comment:
For this and all the FreeLook scenes: new input controls are not set up.  You need to install both new and legacy inputs and reset the component.

![image](https://user-images.githubusercontent.com/6424658/205768108-76cad297-005d-42bc-b21d-979cdb191d4d.png)

It should look like this:
![image](https://user-images.githubusercontent.com/6424658/205768169-e2636a93-7f37-4da6-b3bc-443ab3396b69.png)